src/App/ExpressionParser.tab.h

```diff
@@ -30,6 +31,10 @@
    This special exception was added by the Free Software Foundation in
    version 2.2 of Bison.  */
 
+/* DO NOT RELY ON FEATURES THAT ARE NOT DOCUMENTED in the manual,
+   especially those whose name start with YY_ or yy_.  They are
+   private implementation details that can be changed or removed.  */
+
```

Comment:
Is this from GNU Bison, or from author of the PR? If former, we are still using these features and may be an issue. If the latter, could you elaborate why you're choosing this nomenclature?

Reply:
OK I should have read this:
> GNU Bison regenerated these files, but I'm not sure what to do with them. Let me know if > I should do something different with these.
> * src/App/ExpressionParser.tab.c
> * src/App/ExpressionParser.tab.h

As to the question itself, I see there are items like `#line` directives, which may be very easily messed up if edited by hand by as much as a newline. It's best to let these be generated automatically.